Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -43,10 +43,10 @@ open class PieChartRenderer: NSObject, DataRenderer
         // If we redraw the data, remove and repopulate accessible elements to update label values and frames
         accessibleChartElements.removeAll()
 
-        for case let set as PieChartDataSetProtocol in pieData where
-            set.isVisible && set.entryCount > 0
+        for case let dataSet as PieChartDataSetProtocol in pieData where
```

Comment:
Some inconsistency if the `as ChartData)` cast is necessary or not.  We have it for Candle and Radar but not for Pie chart?

Reply:
@joeboyscout04 This commit was cherry-picked from a different user, so i would rather keep it as is